components/molecules/SearchSelect.jsx

```diff
@@ -0,0 +1,185 @@
+import React, { Component } from 'react';
```

Comment:
i know this is based on the DS component, but i always found it confusing, because when you "select" something it doesn't look like you have chosen the option, and you can delete the text completely and think it's not still selected.

on some apps i've done other variations, like replacing it with a tag, or putting a tag within the input. or on ad manager we clear the input whenever something is selected and someone changes the search term input in any way. what do you think?